Serilog.Sinks.Network/NetworkLoggerConfigurationExtensions.cs

```diff
@@ -68,6 +68,28 @@ public static class NetworkLoggerConfigurationExtensions
             return loggerConfiguration.Sink(sink, restrictedToMinimumLevel);
         }
 
+        public static LoggerConfiguration FluentdTCPSink(this LoggerSinkConfiguration loggerSinkConfiguration, string ip, int port, string defaultTag)
+        {
+            return loggerSinkConfiguration.TCPSink($"tcp://{ip}", port, new FluentdJsonFormatter(defaultTag));
+        }
+
+        public static LoggerConfiguration FluentdUDPSink(this LoggerSinkConfiguration loggerSinkConfiguration, string ip, int port, string defaultTag)
+        {
+            return loggerSinkConfiguration.UDPSink($"udp://{ip}", port, new FluentdJsonFormatter(defaultTag));
+        }
+
+        public static long ToEpochTime(this DateTimeOffset offset)

```

Comment:
Use this https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.tounixtimeseconds?view=net-6.0